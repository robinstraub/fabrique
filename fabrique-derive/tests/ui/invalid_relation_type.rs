use fabrique_derive::Factory;

#[derive(Factory)]
struct Anvil {
    #[fabrique(relation = "Not A Valid Type", referenced_key = "id")]
    hammer_id: u32,
    weight: u32,
}

fn main() {}