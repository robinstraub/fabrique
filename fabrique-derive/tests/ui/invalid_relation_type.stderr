error: Unknown value: `Not A Valid Type` at relation
 --> tests/ui/invalid_relation_type.rs:4:1
  |
4 | struct Anvil {
  | ^^^^^^
